symFileManager.py

```diff
@@ -57,30 +158,30 @@ def GetLibSymbolMap(self, libName, breakpadId):
     if libSymbolMap is None:
       LogDebug("Need to fetch PDB file for " + libName + " " + breakpadId)
 
-      # Guess the name of the .sym file on disk
-      if libName[-4:] == ".pdb":
-        symFileName = re.sub(r"\.[^\.]+$", ".sym", libName)
-      else:
-        symFileName = libName + ".sym"
-
-      pathSuffix = os.path.join(libName, breakpadId, symFileName)
-      urlSuffix = "/".join([libName, breakpadId, symFileName])
+      libSymbolMap = self.diskCache.fetch(pathSuffix)
 
       # Look in the symbol dirs for this .sym file
-      for symbolPath in self.sOptions["symbolPaths"]:
-        path = os.path.join(symbolPath, pathSuffix)
-        libSymbolMap = self.FetchSymbolsFromFile(path)
-        if libSymbolMap:
-          break
-
-      # If not in symbolPaths try URLs
       if not libSymbolMap:
-        for symbolURL in self.sOptions["symbolURLs"]:
-          url = urlparse.urljoin(symbolURL, urlSuffix)
-          libSymbolMap = self.FetchSymbolsFromURL(url)
+        LogDebug("Not found in disk cache, search in local symbol paths.")
+        for symbolPath in self.sOptions["symbolPaths"]:
+          path = os.path.join(symbolPath, pathSuffix)
+          libSymbolMap = self.FetchSymbolsFromFile(path)
           if libSymbolMap:
             break
 
+        # If not in symbolPaths try URLs
+        if not libSymbolMap:
+          LogDebug("Not found in local paths, search symbols in remote URL.")
+          for symbolURL in self.sOptions["symbolURLs"]:
+            url = urlparse.urljoin(symbolURL, urlSuffix)
+            libSymbolMap = self.FetchSymbolsFromURL(url)
+            if libSymbolMap:
+              break
+
+        if libSymbolMap:
```

Comment:
indentation is getting a bit much in this method, can you reduce it by splitting this into 2 methods and returning libSymbolMap as soon as it's found